core/shared/src/main/scala/org/typelevel/idna4s/core/package.scala

```diff
@@ -108,7 +108,7 @@ package object core {
     buffer
   }
 
-  final private[this] val HalfIntMaxValue = Int.MaxValue / 2
+  final private[this] val HalfIntMaxValue: Int = Int.MaxValue / 2
```

Comment:
Why add the type annotation? This prevents it from inlining.

Reply:
Ah indeed. Either Scala.js or scalafix requires explicit result type.

Reply:
Hmm, scalafix bug? 🤔

Reply:
Could be. I've disabled scalafix for this line as a quick fix.